integration_tests/cork_test.go

```diff
@@ -94,7 +93,7 @@ func (s *IntegrationTestSuite) TestCork() {
 
 			// make it rain
 			for i := range s.chain.stewards {
-				addr := fmt.Sprintf("localhost:%v", port+i)
+				addr := fmt.Sprintf("localhost:%v", 5734+i)
```

Comment:
hrm I guess I left this as a magic number everywhere else. Would be good to put it in a var block in setup_tests.go, but it's whatevs.